java-checks/src/test/java/org/sonar/java/checks/unused/UnusedPrivateMethodCheckTest.java

```diff
@@ -40,7 +40,15 @@ void test_non_compiling() {
     CheckVerifier.newVerifier()
       .onFile(TestUtils.nonCompilingTestSourcesPath("checks/UnusedPrivateMethod.java"))
       .withCheck(new UnusedPrivateMethodCheck())
-      .verifyNoIssues();
+      .verifyIssues();
+  }
+  
+  @Test
+  void test_non_compiling_unknown() {
+    CheckVerifier.newVerifier()
+      .onFile(TestUtils.nonCompilingTestSourcesPath("checks/unused/UnusedPrivateMethod.java"))
```

Comment:
Can you rename the file so it does not have the same as the other before? For instance
`UnusedPrivateMethodWithUnknownResolution.java`